activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ModelServiceImpl.java

```diff
@@ -381,11 +381,10 @@ public Optional<ModelContent> createModelContentFromModel(Model model, FileConte
     @Override
     public Model importSingleModel(Project project, ModelType modelType, FileContent fileContent) {
         ImportedModel importedModel = importModel(project, modelType, fileContent);
-        return updateModelContent(
-            importedModel.getModel(),
-            fileContent,
-            Map.of(importedModel.getOrignialId(), importedModel.getUpdatedId())
-        );
+        Map<String, String> identifiersToUpdate = importedModel.hasIdentifiersToUpdate()
+            ? Map.of(importedModel.getOrignialId(), importedModel.getUpdatedId())
```

Comment:
I'm noticing this typo only now on `getOrignialId` (should be `getOriginalId`)

Reply:
Good catch. I'll fix it.